admin/src/main/java/com/exadel/frs/service/UserService.java

```diff
@@ -67,6 +68,10 @@
     private final EmailSender emailSender;
     private final Environment env;
     private final AuthorizationManager authManager;
+    private final StatisticsDatabaseClient statisticsDatabaseClient;
+
+    @Value("${app.feign.appery-io.database.id}")
+    private String statisticsDatabaseId;
```

Comment:
Please rename `db id` to `api key` everywhere

Reply:
Renamed to app.feign.appery-io.api-key